### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by tsregit**
## Description
 - Add source with navbar.
 - Add source with footer.

## Screenshot
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,36 +59,43 @@
...
-          <p class="text-big m-bottom-1">Django</p>
+          <p class="text-medium m-bottom-1">Django</p>
           <p class="text-medium m-bottom-2">Release: 2005</p>
           <p class="text-small">Django(JANG - goh) is a free open-source web framework, written in Phyton, which follows the model-view-template(MVT) architectural pattern.</p>
       </div>
     </div>
     </div>
   </section>
+<footer class="wrapper-footer">
+<div class="container-footer p-top-3">
```
Comment (arconejeros) [inline index.html]: check the identation here

File: scss/footer.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  display: flex;
+  background-color: #00ADEE;
+  flex-direction: column;
+  align-items: center;
+}
+
+.wrapper-footer {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (arconejeros) [inline scss/footer.scss]: add an empty line after this

File: scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.container-navbar {
+  position: fixed;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  -webkit-box-shadow: 0 7px 30px -5px rgba(0, 0, 0, 0.30);
+  -moz-box-shadow: 0 7px 30px -5px rgba(0, 0, 0, 0.30);
+  box-shadow: 0 7px 30px -5px rgba(0, 0, 0, 0.30);
+  z-index: 1000;
```
Comment (arconejeros) [inline scss/navbar.scss]: i think with one or two is sufficient

File: index.html [hunk 4]
```diff
@@ -58,36 +59,43 @@
   <section>
     <div class="title-decoration m-top-10 m-bottom-3">
       <img class="title-image" src="triangle.svg" alt="Front-End Tools"/>
-      <p class="title-text">Back-End Tools</p>
+      <p class="title-text text-big">Back-End Tools</p>
     </div>
-    <div class="wrapper-tech">
+    <div class="wrapper-tech m-bottom-3">
         <div class="container-tech">
         <div class="tech">
```
Comment (arconejeros) [inline index.html]: check the identation

File: scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: #FFF;
```
Comment (arconejeros) [inline scss/navbar.scss]: use scss variables

File: scss/styles.scss [hunk 6]
```diff
@@ -2,4 +2,60 @@
...
+
+.text-medium {
+  font-size: 16px;
+  display: block;
+}
 
+.text-small {
+  font-size: 12px;
+  display: block;
+}
```
Comment (arconejeros) [inline scss/styles.scss]: an empty line should be at the end of every files

File: scss/footer.scss (hunk 2 above)
Comment (tsregit) [reply scss/footer.scss]: the change it is not reflected.

File: scss/footer.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  color: white;
+}
+
+.color-text-footer:nth-child(3) {
+  align-self: flex-end;
+}
+
+.container-footer {
+  display: flex;
+  background-color: #00ADEE;
```
Comment (gon-escalante) [inline scss/footer.scss]: move this to `colors.scss`

File: scss/navbar.scss [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+  width: 200px;
+  margin-left: 20px;
+}
+
+.container-navbar {
+  position: fixed;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  -webkit-box-shadow: 0 7px 30px -5px rgba(0, 0, 0, 0.30);
```
Comment (gon-escalante) [inline scss/navbar.scss]: id also move this `rgba(0, 0, 0, 0.30)` to `colors.scss`

**Review by gon-escalante — CHANGES_REQUESTED** [review]

**Review by gon-escalante — APPROVED** [review]

**Pull request merged**